src/smolagents/agents.py

```diff
@@ -647,30 +651,28 @@ def planning_step(self, task, is_first_step: bool, step: int) -> None:
         if is_first_step:
             message_prompt_facts = {
                 "role": MessageRole.SYSTEM,
-                "content": SYSTEM_PROMPT_FACTS,
+                "content": wrap_content_text(SYSTEM_PROMPT_FACTS),
```

Comment:
i'd prefer to just use the plain syntax:
```suggestion
                "content": [{"type": "text", "text": SYSTEM_PROMPT_FACTS}],
```